src/Sulu/Bundle/AdminBundle/Gruntfile.js

```diff
@@ -137,10 +153,16 @@ module.exports = function(grunt) {
             },
             buildMain: {
                 options: {
-                    variables: {
-                        '/js/main.js': '/dist/main.js'
-                    },
-                    prefix: ''
+                    patterns: [
+                        {
+                            match: /\(new Date\)\.getTime\(\)/,
+                            replacement: time.toString()
+                        },
+                        {
+                            match: /\/js\/main\.js/,
+                            replacement: '/dist/main.js'
```

Comment:
Any reason you are sometimes escaping the characters and sometimes are using quotes?